.changeset/moody-melons-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-react': minor
+---
+
+Add Microsoft Entra ID auth provider supporting custom Enterprise App scope
```

Comment:
Can we make sure to add some docs for this, somewhere here could work: https://backstage.io/docs/features/kubernetes/configuration